Public/Src/Sandbox/MacOs/Interop/Posix/memory.c

```diff
@@ -42,15 +42,58 @@ int GetRamUsageInfo(RamUsageInfo *buffer, long bufferSize)
     return KERN_SUCCESS;
 }
 
+static uint64_t ProcessTreeWorkingSetSize(pid_t pid, const rlim_t max_proc_count, bool *success)
```

Comment:
is `proc_pid_rusage` ever called for this `pid`?  It looks like it only queries its children and for each of them called `proc_pid_rusage`, but not for this root process itself